index.js

```diff
@@ -112,6 +122,43 @@ class Conf {
 		}
 	}
 
+	_migrate(migrations = {}, versionToMigrate = '0.0.0') {
+		const MIGRATION_KEY = '__conf-migrated-version__';
+
+		const previousMigratedVersion = this.get(MIGRATION_KEY, '0.0.0');
+
+		const olderVersionsWithMigrations = Object.keys(migrations)
```

Comment:
Nifty pick: perhaps a better name for this variable should be `olderVersions` or `migrationVersions` as it contains an array of versions that need to be migrated?

Reply:
I think I should actually call it newerVersions since they actually represent a migration with a version bigger than the current one 🤔